src/xtgeo/grid3d/grid_properties.py

```diff
@@ -206,11 +435,13 @@ def dates(self):
             ...     print ('Date applied is {}'.format(date))
             Date applied is 19991201
 
+        .. versionchanged:: 2.16 dates is no longer an alias (undocumented behavior),
+            and simply return the date of the underlying GridProperty.
```

Comment:
_"...simply return dates of the underlying list of GridProperty"_ ?

Reply:
Fixed